Lib/test/test_pathlib.py

```diff
@@ -2587,6 +2620,32 @@ def test_handling_bad_descriptor(self):
 class WindowsPathTest(_BasePathTest, unittest.TestCase):
     cls = pathlib.WindowsPath
 
+    def test_absolute(self):
+        P = self.cls
+
+        # Simple absolute paths
+        self.assertEqualNormCase(str(P('c:\\').absolute()), 'c:\\')
```

Comment:
I was going to suggest using `r'c:\a\b'` to get more readable strings, but realized it doesn't work with a trailing backslash `r'c:\'` . So it is fine to use regular string literals.

Reply:
Yeah. `r''` is great for regexs but surprisingly limited when you apply it to Windows paths...